gdbinit.py

```diff
@@ -34,7 +42,13 @@ def calculate_hash(file_path):
 
 
 def run_poetry_install(dev=False):
-    command = ["poetry", "install"]
+    check_poetry()  # Check if poetry executable exists in PATH
+    poetry_path = shutil.which("poetry")
+    if poetry_path is None:
+        print("Poetry is not installed. Please install it and try again.")
+        sys.exit(1)
```

Comment:
Let's not exit, let's just return. The message we print is useful so the user knows there's a configuration issue, but the issue isn't a fatal error so we should continue.

Reply:
okay got it.